L1Trigger/L1TTrackMatch/python/L1GTTInputProducer_cfi.py

```diff
@@ -1,13 +1,13 @@
 import FWCore.ParameterSet.Config as cms
 
-L1GTTInputProducer = cms.EDProducer('L1GTTInputProducer',
-  l1TracksInputTag = cms.InputTag("TTTracksFromTrackletEmulation", "Level1TTTracks"),
+l1tGTTInputProducer = cms.EDProducer('L1GTTInputProducer',
```

Comment:
Duplicate of
cfipython/L1Trigger/L1TTrackMatch/l1GTTInputProducer_cfi.py
with the same search path

Reply:
Done